src/core/Akka/Util/Reflection/TypeCache.cs

```diff
@@ -15,7 +16,7 @@ namespace Akka.Util.Reflection
     /// </summary>
     public static class TypeCache
     {
-        private static readonly ConcurrentDictionary<string, Type> TypeMap = new ConcurrentDictionary<string, Type>();
+        private static readonly ConcurrentDictionary<string, Type> TypeMap = new ConcurrentDictionary<string, Type>(new[] { new KeyValuePair<string, Type>("null", null),  });
```

Comment:
This is a hack, but it avoids having to add a `str.Equals("null")` check for literally every type.